helm-vote/values.yaml

```diff
@@ -9,6 +9,11 @@ namespace: default
 clusterIP: 10.10.0.10
 clusterDomain: nip.io
 registry: &defaultRegistry local-registry:5002
+ballot_option: "{{option}}"
+ecserver_name: "{{name}}"
+ecserver_image_url: "{{image_url}}"
+ballot_ip: "http://default.ballot.10.10.0.10.nip.io"
```

Comment:
ballot_ip and ecserver_ip is not needed
Rather it can be derived using existing values
example:
image: '{{- printf "%s/%s:%s" .Values.registry .Values.services.ballot.imageName .Values.tag -}}'